data/zooroids/zooroids.txt

```diff
@@ -0,0 +1,168 @@
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+ship "Zooroid"
+	noun creature
+	sprite "ship/zooroids/zooroid-a"
+		"frame rate" 24
+		"random start frame"
+	swizzle 0
+	"never disabled"
+	attributes
+		"category" "Fighter"
+		"cost" 0
+		"hull" 400
+		"required crew" 1
+		"bunks" 1
+		"mass" 60
+		"drag" 8
+		"heat dissipation" 1.4
+		"cargo space" 0
+		"outfit space" 0
+		"weapon capacity" 0
+		"engine capacity" 0
+		"hull repair rate" 1
+		"inscrutable" 1
+		"energy capacity" 100
+		"energy generation" 1
+		"thrust" 10
+		"turn" 120
+		"energy protection" 1
+		"ion protection" 1
+	"leak" "blood" 1 2
+	"final explode" "blood"
+	description `These are tiny space creatures that feed on solar energy, storing it into two receptors found at the front of their bodies. They are extremely docile, and are found in large groups - almost never seen alone.`
+	
+
+
+ship "Zooroid" "Zooroid (B)"
+	noun creature
+	sprite "ship/zooroids/zooroid-b"
+		"frame rate" 24
+		"random start frame"
+	swizzle 0
+	"never disabled"
+	attributes
+		"category" "Fighter"
+		"cost" 0
+		"hull" 1420
+		"required crew" 1
+		"bunks" 1
+		"mass" 140
+		"drag" 10
+		"heat dissipation" 1
+		"cargo space" 0
+		"outfit space" 0
+		"weapon capacity" 0
+		"engine capacity" 0
+		"hull repair rate" 1
+		"inscrutable" 1
+		"energy capacity" 100
+		"energy generation" 1
+		"thrust" 12
+		"turn" 60
+		"energy protection" 1
+		"ion protection" 1
+	"leak" "blood" 2 3
+	"final explode" "blood"
+	description `Oddly shaped like a boomerang, these space creatures are in a constant rotary motion caused by a small internal organ. The motion provides a natural defense against small debris and projectiles that threaten to intercept the creature, which is harmless otherwise.`
+
+ship "Zooroid" "Zooroid (C)"
+	sprite "ship/zooroids/zooroid-c"
+		"frame rate" 24
+		"random start frame"
+	swizzle 0
+	"never disabled"
+	attributes
+		category "Fighter"
+		"cost" 0
+		"hull" 860
+		"required crew" 1
+		"bunks" 1
+		"mass" 200
+		"drag" 9
+		"heat dissipation" 1.8
+		"fuel capacity" 0
+		"cargo space" 0
+		"outfit space" 0
+		"weapon capacity" 0
+		"engine capacity" 0
+		"energy capacity" 1000
+		"energty generation" 100
+		"thrust" 6
+		"turn" 70
+		"energy protection" 1
+		"heat protection" 1
+		weapon
+			"blast radius" 1600
+			"shield damage" 3000
+			"hull damage" 8000
+			"hit force" 4000
+			"piercing" 0.1
+	engine 0 0
+	explode "flamethrower die" 1
+	"final explode" "fission explosion" 
+	description `This species of space life is found around hotter stars and feeds on solar energy, storing it in four organs that give off the appearance of eyes. These creatures are however easily agitated, and explode violently when provoked, sometimes seen doing so aggressively.`
```

Comment:
If the Zooroid explodes in defense, is this a suicidal measure like bee stings or can the Zooroid survive? If it's suicidal, then what's the evolutionary purpose behind this defense mechanism?

Reply:
As it turns out, I can't make them self-destruct on purpose anyways. I'll omit that part until someday in the future we can do the behavior.